javascript/apps/taiga/src/app/modules/workspace/feature-list/components/workspace-item/workspace-item.component.html

```diff
@@ -95,7 +86,7 @@
       <tg-project-card
         *ngIf="
           workspace.latestProjects.length === 0 &&
-          workspace.userRole === 'admin' &&
+          workspace.userRole === 'member' &&
```

Comment:
When the workspace is empty it should show a project card placeholder
![Screenshot from 2023-06-07 10-30-29](https://github.com/taigaio/taiga/assets/5970959/9b3c1797-e091-43c5-9a00-e9ffa29ffb39)

Reply:
'Project button' is missing also

Reply:
Endpoint 
http://localhost:8000/api/v2/my/workspaces is returning
```
[{"id":"QTya1gUXEe67nA3q1DAN0g","name":"new wks","slug":"new-wks","color":2,"latestProjects":[],"invitedProjects":[],"totalProjects":0,"hasProjects":false,"userRole":"admin"},{"id":"wh8bVgUVEe67nA3q1DAN0g","name":"exploit distributed methodologies","slug":"exploit-distributed-methodologies","color":2,"latestProjects":[{"logo":"project/5/3/3/3/d5304a7541243e28a2f3fd28a33f20656ff92ab243470036ff621d37f912/logo-019.png","logoSmall":"http://localhost:8000/media/project/5/3/3/3/d5304a7541243e28a2f3fd28a33f20656ff92ab243470036ff621d37f912/logo-019.png.32x32_q85_crop.png","logoLarge":"http://localhost:8000/media/project/5/3/3/3/d5304a7541243e28a2f3fd28a33f20656ff92ab243470036ff621d37f912/logo-019.png.80x80_q85_crop.png","id":"wh8cVAUVEe67nA3q1DAN0g","name":"Polarized web-enabled database","slug":"polarized-web-enabled-database","description":"Store last left.","color":5}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bVAUVEe67nA3q1DAN0g","name":"innovate distributed channels","slug":"innovate-distributed-channels","color":7,"latestProjects":[{"logo":null,"logoSmall":null,"logoLarge":null,"id":"wh8cQgUVEe67nA3q1DAN0g","name":"Total local initiative","slug":"total-local-initiative","description":"It easy law view central leg.","color":2}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bUgUVEe67nA3q1DAN0g","name":"re-intermediate cross-platform expe","slug":"re-intermediate-cross-platform-expe","color":2,"latestProjects":[{"logo":null,"logoSmall":null,"logoLarge":null,"id":"wh8cMAUVEe67nA3q1DAN0g","name":"Configurable dynamic intranet","slug":"configurable-dynamic-intranet","description":"Pm more heavy across while top. Daughter fall likely wear someone everybody.","color":3}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bUAUVEe67nA3q1DAN0g","name":"matrix leading-edge e-business","slug":"matrix-leading-edge-e-business","color":8,"latestProjects":[{"logo":null,"logoSmall":null,"logoLarge":null,"id":"wh8cHgUVEe67nA3q1DAN0g","name":"Operative stable analyzer","slug":"operative-stable-analyzer","description":"After back available inside nearly scientist central instead.","color":6}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bTgUVEe67nA3q1DAN0g","name":"aggregate integrated models","slug":"aggregate-integrated-models","color":2,"latestProjects":[{"logo":"project/6/4/9/1/70d68ffc799ec28b668ea860ce5ce578683a54c221c921714fa9699b6f39/logo-007.png","logoSmall":"http://localhost:8000/media/project/6/4/9/1/70d68ffc799ec28b668ea860ce5ce578683a54c221c921714fa9699b6f39/logo-007.png.32x32_q85_crop.png","logoLarge":"http://localhost:8000/media/project/6/4/9/1/70d68ffc799ec28b668ea860ce5ce578683a54c221c921714fa9699b6f39/logo-007.png.80x80_q85_crop.png","id":"wh8cDAUVEe67nA3q1DAN0g","name":"Implemented explicit neural-net","slug":"implemented-explicit-neural-net","description":"Or ball mouth film. Today watch source firm drug senior.","color":6}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bTAUVEe67nA3q1DAN0g","name":"whiteboard world-class platforms","slug":"whiteboard-world-class-platforms","color":6,"latestProjects":[{"logo":"project/4/6/9/7/55de6afdc5156ef2b20e2a1116819a8fc60db02e677c8f977cc707c3da4d/logo-011.png","logoSmall":"http://localhost:8000/media/project/4/6/9/7/55de6afdc5156ef2b20e2a1116819a8fc60db02e677c8f977cc707c3da4d/logo-011.png.32x32_q85_crop.png","logoLarge":"http://localhost:8000/media/project/4/6/9/7/55de6afdc5156ef2b20e2a1116819a8fc60db02e677c8f977cc707c3da4d/logo-011.png.80x80_q85_crop.png","id":"wh8b-gUVEe67nA3q1DAN0g","name":"Implemented holistic solution","slug":"implemented-holistic-solution","description":"Do center law live religious election agreement decade.","color":8}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bSgUVEe67nA3q1DAN0g","name":"utilize robust metrics","slug":"utilize-robust-metrics","color":3,"latestProjects":[{"logo":"project/4/8/1/1/90aa23860b5cf6f270d14e8ff19ad2eff2d590ece88ac970b208b50621a6/logo-022.png","logoSmall":"http://localhost:8000/media/project/4/8/1/1/90aa23860b5cf6f270d14e8ff19ad2eff2d590ece88ac970b208b50621a6/logo-022.png.32x32_q85_crop.png","logoLarge":"http://localhost:8000/media/project/4/8/1/1/90aa23860b5cf6f270d14e8ff19ad2eff2d590ece88ac970b208b50621a6/logo-022.png.80x80_q85_crop.png","id":"wh8b6AUVEe67nA3q1DAN0g","name":"Team-oriented 4thgeneration hardware","slug":"team-oriented-4thgeneration-hardware","description":"Act window standard audience. Debate daughter purpose voice.","color":1}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bSAUVEe67nA3q1DAN0g","name":"streamline mission-critical info-me","slug":"streamline-mission-critical-info-me","color":3,"latestProjects":[{"logo":"project/a/f/a/3/184235f44ebdd8d14eb6eb863f4752a207a51971deacaeac8e059e4b6a99/logo-004.png","logoSmall":"http://localhost:8000/media/project/a/f/a/3/184235f44ebdd8d14eb6eb863f4752a207a51971deacaeac8e059e4b6a99/logo-004.png.32x32_q85_crop.png","logoLarge":"http://localhost:8000/media/project/a/f/a/3/184235f44ebdd8d14eb6eb863f4752a207a51971deacaeac8e059e4b6a99/logo-004.png.80x80_q85_crop.png","id":"wh8b1gUVEe67nA3q1DAN0g","name":"Vision-oriented coherent superstructure","slug":"vision-oriented-coherent-superstructure","description":"Issue indicate market ten foot education good.","color":6}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bRgUVEe67nA3q1DAN0g","name":"re-intermediate plug-and-play archi","slug":"re-intermediate-plug-and-play-archi","color":8,"latestProjects":[{"logo":"project/9/7/7/b/ea640e22ea70e7dce6311c075c2e18b9e423e3aefb710f3a870aa85a32eb/logo-015.png","logoSmall":"http://localhost:8000/media/project/9/7/7/b/ea640e22ea70e7dce6311c075c2e18b9e423e3aefb710f3a870aa85a32eb/logo-015.png.32x32_q85_crop.png","logoLarge":"http://localhost:8000/media/project/9/7/7/b/ea640e22ea70e7dce6311c075c2e18b9e423e3aefb710f3a870aa85a32eb/logo-015.png.80x80_q85_crop.png","id":"wh8bxAUVEe67nA3q1DAN0g","name":"Function-based 4thgeneration model","slug":"function-based-4thgeneration-model","description":"Then director simply those physical maybe model. Figure box international not type very indeed.","color":7}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wh8bRAUVEe67nA3q1DAN0g","name":"exploit synergistic e-markets","slug":"exploit-synergistic-e-markets","color":1,"latestProjects":[{"logo":"project/0/7/c/5/73460b82f132bdbc3d68874b96c8ea5480453e0180cebbe45c9ce988dcdb/logo-020.png","logoSmall":"http://localhost:8000/media/project/0/7/c/5/73460b82f132bdbc3d68874b96c8ea5480453e0180cebbe45c9ce988dcdb/logo-020.png.32x32_q85_crop.png","logoLarge":"http://localhost:8000/media/project/0/7/c/5/73460b82f132bdbc3d68874b96c8ea5480453e0180cebbe45c9ce988dcdb/logo-020.png.80x80_q85_crop.png","id":"wh8bsgUVEe67nA3q1DAN0g","name":"Programmable zero tolerance core","slug":"programmable-zero-tolerance-core","description":"Know series lay smile away and.","color":8}],"invitedProjects":[],"totalProjects":1,"hasProjects":true,"userRole":"admin"},{"id":"wxA2DgUVEe67nA3q1DAN0g","name":"Custom workspace","slug":"custom-workspace","color":7,"latestProjects":[{"logo":"project/7/e/2/8/e1080569eb402512897a222f9563bb89229d833c14120dc01c914b7a40d3/logo-008.png","logoSmall":"http://localhost:8000/media/project/7/e/2/8/e1080569eb402512897a222f9563bb89229d833c14120dc01c914b7a40d3/logo-008.png.32x32_q85_crop.png","logoLarge":"http://localhost:8000/media/project/7/e/2/8/e1080569eb402512897a222f9563bb89229d833c14120dc01c914b7a40d3/logo-008.png.80x80_q85_crop.png","id":"yaDhLAUVEe67nA3q1DAN0g","name":"2k Stories, 40 statuses","slug":"2k-stories-40-statuses","description":"This project contains 2000 stories and 40 statuses.","color":4},{"logo":null,"logoSmall":null,"logoLarge":null,"id":"xX6WkwUVEe67nA3q1DAN0g","name":"1k Stories","slug":"1k-stories","description":"This project contains 1000 stories.","color":6},{"logo":null,"logoSmall":null,"logoLarge":null,"id":"wxA2IgUVEe67nA3q1DAN0g","name":"Several Roles","slug":"several-roles","description":"Fire maintain worry statement much election. Campaign newspaper personal really mean either college.","color":5}],"invitedProjects":[],"totalProjects":3,"hasProjects":true,"userRole":"guest"}]
```

It should be 'member' insteand of 'admin' the userRole